### DISCUSSION TITLE
Frunk and trunk as cover

### DISCUSSION THREAD

**Discussion by folrewef** [alandtse/tesla#784] [General]
Why is frunk and trunk set as cover? I cannot open them. Would be great to be able to open them.

**Comment by skipishere** [comment]
Not sure what you question is. I've just tested the trunk/boot on my Model 3 using this and it works both for opening and closing.
I assume the Frunk works the same but given I have to manually close (thanks Tesla) I can't test it at this moment.

Can you supply more details about what the issue is, can you open and close your trunk in the Tesla app, and similar open the Frunk in the Tesla app?

**Reply by folrewef** [reply]
I have an older version where I need to open and close both frunk and trunk manually. SDo my need is to unlock not to open and close.

**Reply by folrewef** [reply]
I cannot unlock any of them.

**Reply by folrewef** [reply]
I am sorry, I actually did get both of them to work. I geuss I did not check that the car was connected.

**Reply by folrewef** [reply]
But then now I tried again and it didn't work. I tried blink and it worked so there was a connection. I tried with the car locked and unlocked - no difference.

**Reply by skipishere** [reply]
Is this behaviour different from the Tesla app vs doing it with this integration? Also keep in mind that if the car is in sleep mode there can be a few seconds delay between raising the event and the car actioning it (catches me out sometimes unlocking the doors  🤦)

**Reply by folrewef** [reply]
I will check app vs integration

**Reply by folrewef** [reply]
It is strange. I went down and tried it again. It worked - both frunk and trunk. I closed them both manually and in the app (they seem not to react to being closed). Maybe the problem is that they need to updated from the Tesla app to recognize that they are closed.
